tests/slice_by_index_list_test.py

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
There are no tests that slice in 'yy'. Please add one (or change an existing one) to make sure that slicing works regardless of whether it is n inner or outer dim.